sequence_task_1/student/include/impl.h

```diff
@@ -74,8 +74,7 @@ class StaticPlaylist
     template<class... Args>
     const Song& play(Args&&... songData)
     {
-        m_tracklist.insert(m_tracklist.end(), {Song(songData)...});
-        return *prev(m_tracklist.end());
+        play(songData...);
```

Comment:
Use std::forward for universal references
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f19-for-forward-parameters-pass-by-tp-and-only-stdforward-the-parameter